### PR TITLE
Add docs for Sure Petcare set_lock_state service

### PR DESCRIPTION
**Pull request by insertjokehere**
## Proposed change

Add docs for Sure Petcare integration to describe the set_lock_state service added in home-assistant/core/44557

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/44557

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/surepetcare.markdown [hunk 1]
```diff
@@ -55,6 +55,21 @@ surepetcare:
     type: time
 {% endconfiguration %}
 
+## set_lock_state Service
```
Comment (frenck) [inline source/_integrations/surepetcare.markdown]:
```suggestion
## Services

### Service `surepetcare.set_lock_state`
```

File: source/_integrations/surepetcare.markdown [hunk 2]
```diff
@@ -55,6 +55,21 @@ surepetcare:
...
+
+This service let you change the locking state of a flap
+
+| Service data attribute | Required | Type | Description |
+| ---------------------- | -------- | -------- | ----------- |
+| `flap_id` | `True` | integer | Flap ID to change - see below for instructions on finding device IDs
```
Comment (frenck) [inline source/_integrations/surepetcare.markdown:66]: Why isn't this tied to the entity_ids the integration provides?

File: source/_integrations/surepetcare.markdown [hunk 3]
```diff
@@ -55,6 +55,21 @@ surepetcare:
...
+| `lock_state` | `True` | string | New state to change the flap to
+
+`lock_state` should be one of:
+* `unlocked` - flap is unlocked, pets are allowed both in and out
+* `locked_in` - flap is 'in only' - pets can come in but not go back out
+* `locked_out` - flap is 'out only' - pets can go out, but not back in
+* `locked_all` - flap is locked both ways
```
Comment (frenck) [inline source/_integrations/surepetcare.markdown]:
```suggestion

- `unlocked` - flap is unlocked, pets are allowed both in and out.
- `locked_in` - flap is 'in only' - pets can come in but not go back out.
- `locked_out` - flap is 'out only' - pets can go out, but not back in.
- `locked_all` - flap is locked both ways.
```

File: source/_integrations/surepetcare.markdown [hunk 4]
```diff
@@ -55,6 +55,21 @@ surepetcare:
     type: time
 {% endconfiguration %}
 
+## set_lock_state Service
+
+This service let you change the locking state of a flap
```
Comment (frenck) [inline source/_integrations/surepetcare.markdown]:
```suggestion
This service let you change the locking state of a flap.
```

**Review by frenck — CHANGES_REQUESTED** [review]
Thanks for the PR @insertjokehere 👍 

I've reviewed it and left some comments. Could you please take a look? Thanks! 👍

File: source/_integrations/surepetcare.markdown (hunk 4 above)
Comment (insertjokehere) [reply source/_integrations/surepetcare.markdown]: Incorporated suggestion, and fixed 'This service let you'

File: source/_integrations/surepetcare.markdown (hunk 2 above)
Comment (insertjokehere) [reply source/_integrations/surepetcare.markdown:66]: Yeah, I was unsure about the best way to approach this. There isn't really a natural entity to target - the closest we've got is the sensor that reflects the current lock state. It feels a bit weird/unintuitive to me to need to specify a sensor entity, when what I really want to act on is the flap itself. Requiring a user to know the flap ID isn't an especially bad UX, they already need to know it to configure the integration.

If this goes against established patterns, I'll happily rework the PR to fit.

**Comment by insertjokehere** [comment]
@frenck thanks for the review. The PR to add this feature can't merge until after the Python 3.7 deprecation work is complete, so I've marked this as draft for now.

**Review by frenck — APPROVED** [review]
Thanks, @insertjokehere 👍 

Happy New Year 🥂

**Pull request merged**